src/NuGet.Services.AzureSearch/BlobContainerBuilder.cs

```diff
@@ -84,9 +84,9 @@ public async Task CreateAsync(bool retryOnConflict)
             _logger.LogInformation("Done creating blob container {ContainerName}.", _options.Value.StorageContainer);
         }
 
-        public async Task CreateIfNotExistsAsync()
+        public async Task CreateIfNotExistsAsync(BlobRequestOptions options = null, OperationContext operationContext = null)
```

Comment:
These parameters are not needed in this context. Null can be passed for both of these on line 89.

Reply:
This should reduce the number of changes greatly.

Reply:
Fixed in https://github.com/NuGet/NuGet.Services.Metadata/pull/531/commits/a43276a228128b0a265e1dcf63f1200fbbd8e79f